src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx

```diff
@@ -40,19 +40,19 @@ Destination settings require specific [capabilities](/docs/accounts/accounts-bil
 
 ## Manage destinations [#manage-destinations]
 
-1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com), click **Alerts & AI**, and in the left nav under **Enrich and Respond**, click **Destinations**. The destinations table shows information about the existing destinations and allows users to enable, disable, and modify.
-2. To add a destination, click the appropriate platform tile. To modify destination settings, click the destination row in the destinations table.
-
 <img
   title="Destinations overview page"
   alt="Destinations overview page."
   src={destinationsOverview}
 />
 
 <figcaption>
-  **one.newrelic.com > Alerts & AI > Destinations**.
+  Go to **one.newrelic.com > Alerts & AI > Destinations**. Use destinations to choose where your notifications are sent.
 </figcaption>
 
+1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com), click **Alerts & AI**, and in the left nav under **Enrich and Respond**, click **Destinations**. The destinations table shows information about the existing destinations and allows users to enable, disable, and modify.
```

Comment:
@paperclypse, I missed this in the original review, but this first step looks like a couple of steps. You might consider this in a future update:

1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com).
2. Click **Alerts & AI**, and in the left nav under **Enrich and Respond**, click **Destinations**.

Also, "allows users" should probably be "allows you."

Reply:
Thanks! Yeah, this was really just about updating the image for the AI team.

Reply:
A valid point for sure.